debian/control

```diff
@@ -1,13 +1,14 @@
-Source: xivo-nginx
+Source: wazo-nginx
 Section: admin
 Priority: extra
 Maintainer: Wazo Maintainers <dev.wazo@gmail.com>
 Build-Depends: debhelper (>= 9)
 Standards-Version: 3.9.6
 
-Package: xivo-nginx
+Package: wazo-nginx
 Architecture: all
 Pre-Depends: dpkg (>= 1.16.1)
+Provides: xivo-nginx
```

Comment:
You need two more keys: Replaces and Conflicts, so that replacement is handled correctly by Debian

Reply:
ok